### ISSUE TITLE
Determine why run_metric_for_selecting_pipelines is taking forever

### ISSUE THREAD

**Issue by emilycantrell** [emilycantrell/stork_oracle_cbs#12]
I ran a job overnight on the CBS heavy server, with 30 grid rows (many of which had small sample sizes and small feature sets). The models were all finished after about 7-8 hours. However, the final run_metric_for_selecting_pipelines step did not finish after about 4 hours of running. 

I tried the following: 
- Decreased n_bootstrap from 10000 to 2 (previously we've only ever used 2), and increased threshold_increment from 0.001 to 0.01 (usually in the past we've used 0.01). Still, run_metric_for_selecting_pipelines hadn't finished after a couple of hours.
- Freed up some memory because the memory was very close to full. Still, run_metric_for_selecting_pipelines hadn't finished after a couple of hours.
- Now I've reduced grid_rows to a single row to see what happens. This is also taking surprisingly long. 

I'm using workers = 4. A next step if the single grid row doesn't finish soon is to try workers = 1. (There a 4 cores on the heavy server. But maybe I shouldn't be using all 4.)

**Comment by emilycantrell** [comment]
Even on just one grid row, run_metric_for_selecting_pipelines didn't finish after over 20 minutes. This is unusual behavior. I will continue debugging.

**Comment by emilycantrell** [comment]
I realized that when I was trying to restrict to just 1 grid row for testing purposes, I did so incorrectly, and was still running on all 30 rows. 

Maybe I am just too impatient.

This has never been a problem on OSSC, so I'm going to close the issue for now wait and see if it's a problem on OSSC. If so, we can reopen the issue and work on it. 

In the meantime, the job got far enough to produce results for every metric that doesn't require a threshold, so I am going to save and export those results.
